styles.scss

```diff
@@ -272,12 +272,12 @@ $color-bg-dark-stickybox: $color-link;
 // (text-only not supported) on screens less wide.
 // https://ethical-ad-client.readthedocs.io/en/latest/#stickybox
 [data-ea-style="stickybox"].loaded {
-  .ea-type-image {
-    z-index: $zindex-styles;
-    position: fixed;
-    bottom: 20px;
-    right: 20px;
+  position: fixed;
```

Comment:
I assume we want a comment in the CSS explaining why it's done this way, so we don't try to reverse it at some point? Definitely shows the need to have some tests for this code..

Reply:
I'll add a comment and we *definitely* need tests.